app/routes/authenticated.ts

```diff
@@ -141,4 +139,22 @@ export default class AuthenticatedRoute extends Route {
 
     this.session.invalidate();
   }
+
+  @action
+  regLastTransitionMonitor() {
```

Comment:
we can rename this to `monitorLastTransition` and event handler can be `registerLastTransition`